resources/functionMap.php

```diff
@@ -949,9 +949,9 @@
 'chunk_split' => ['string', 'str'=>'string', 'chunklen='=>'int', 'ending='=>'string'],
 'class_alias' => ['bool', 'user_class_name'=>'string', 'alias_name'=>'string', 'autoload='=>'bool'],
 'class_exists' => ['bool', 'classname'=>'string', 'autoload='=>'bool'],
-'class_implements' => ['array<string,string>', 'what'=>'object|string', 'autoload='=>'bool'],
-'class_parents' => ['array<string,class-string>', 'instance'=>'object|string', 'autoload='=>'bool'],
-'class_uses' => ['array<string,string>', 'what'=>'object|string', 'autoload='=>'bool'],
+'class_implements' => ['array<string,string>|false', 'what'=>'object|string', 'autoload='=>'bool'],
```

Comment:
is a interface also a `class-string` which can be used instead of `string` ?

Reply:
Yes